catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/data/impl/ContentItemValidator.java

```diff
@@ -26,35 +24,52 @@
  * required by the {@link ddf.catalog.content.StorageProvider}.
  */
 public class ContentItemValidator {
-
     private static final Pattern QUALIFIER_PATTERN = Pattern.compile("\\w[a-zA-Z0-9_\\-]+");
```

Comment:
Also we allow the UUID's to not contain the dashes (and most cases don't) which is how the actual UUID expects the UUID to come in formatted